README.md

```diff
@@ -1,47 +1,70 @@
 # vm-memory
-A library to access virtual machine's physical memory.
 
-For a typical hypervisor, there are seveval components, such as boot loader, virtual device drivers, virtio backend drivers and vhost drivers etc, need to access VM's physical memory. The `vm-memory` crate provides a set of traits to decouple VM memory consumers from VM memory providers. Based on these traits, VM memory consumers could access VM's physical memory without knowing the implementation details of the VM memory provider. Thus hypervisor components based on these traits could be shared and reused by multiple hypervisors.
+A library to access the physical memory of a virtual machine.
+
+In a typical Virtual Machine Monitor (VMM) there are several components, such
+as boot loader, virtual device drivers, virtio backend drivers and vhost
+drivers, that need to access the VM physical memory. The `vm-memory` crate
+provides a set of traits to decouple VM memory consumers from VM memory
+providers. Based on these traits, VM memory consumers can access the physical
+memory of the VM without knowing the implementation details of the VM memory
+provider. Thus VMM components based on these traits can be shared and reused by
+multiple virtualization solutions.
 
 ## Platform Support
+
 - Arch: x86, AMD64, ARM64
 - OS: Linux/Unix/Windows
 
 ## Usage
+
```

Comment:
I don't really like the `Usage` section because we reference a particular version of `vm-memory`, and the `extern crate vm_memory;` declaration is no longer necessary post edition 2018. 

How about we remove it altogether? Adding the `vm-memory` crate to a Rust project is the same as for any other crate, and users supposedly already know how to do that.

Reply:
I removed the section.